ui/client/index.d.ts

```diff
@@ -6,3 +7,24 @@ declare const __GIT__: {
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 declare type $TodoType = any
+
```

Comment:
Why it is added? Type script isn't aware of webpack config? Excuse my ignorance ;)